flang/lib/Lower/OpenMP.cpp

```diff
@@ -159,7 +161,13 @@ static void createBodyOfOp(
     firOpBuilder.createBlock(&op.getRegion(), {}, tiv);
     for (auto &arg : args) {
       fir::ExtendedValue exval = op.getRegion().front().getArgument(argIndex);
-      [[maybe_unused]] bool success = converter.bindSymbol(*arg, exval);
+      auto val = fir::getBase(exval);
```

Comment:
I think you can simply merged these two lines into:
`auto val = op.getRegion().front().getArgument(argIndex);` There is no need to go through a `fir::ExtendedValue` anymore.